tests/test_api/test_export.py

```diff
@@ -1,41 +1,37 @@
-from django.contrib.auth.models import User
-from django.test.client import Client
 from django.urls import reverse
 
+from rest_framework import status, test
+
 import pytest
 
 from import_export_extensions.models import ExportJob
 
 
 @pytest.mark.django_db(transaction=True)
-def test_export_api_creates_export_job(client: Client, superuser: User):
+def test_export_api_creates_export_job(admin_api_client: test.APIClient):
     """Ensure export start API creates new export job."""
-    client.force_login(superuser)
-
-    response = client.post(
+    response = admin_api_client.post(
         path=reverse("export-artist-start"),
         data={
             "file_format": "csv",
         },
     )
+    assert response.status_code == status.HTTP_201_CREATED
     assert response.data["export_status"] == "CREATED"
     assert ExportJob.objects.first()
```

Comment:
Maybe use `id` from response and then `assert ExportJob.objects.filter(id=ID).exists()?`